config/workflows/gisDeliveryWF.xml

```diff
@@ -23,6 +23,10 @@
     <prereq>reset-geowebcache</prereq>
     <label>Finalize delivery workflow for the object</label>
   </process>
+  <process name="reload-geoserver">
+    <prereq>finish-gis-delivery-workflow</prereq>
+    <label>Automatically reload the geo server</label>
+  </process>
   <process name="start-accession-workflow">
     <prereq>finish-gis-delivery-workflow</prereq>
```

Comment:
do you want this step to be sequential after reload-geoserver?    I'm not sure if we have non-sequential workflows.  Maybe.

Reply:
Good catch, thank you.